Multisig/Logic/Models/TokenRegistry/InMemoryTokenStore.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  InMemoryTokenStore.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 13.07.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct InMemoryTokenStore {
+
+    private var items: [Address: Token] = [:]
+
+    func token(address: Address) -> Token? {
+        items[address]
+    }
+
+    mutating func add(_ token: Token) {
```

Comment:
As it's mutable, is it better to use class instead of struct?

Reply:
no need, this is intended, i.e. the store is mutable, like an Array has mutable operations